chain/indexer/src/streamer/mod.rs

```diff
@@ -17,17 +17,27 @@ use near_indexer_primitives::{
 use near_parameters::RuntimeConfig;
 use near_primitives::hash::CryptoHash;
 use near_primitives::views;
-use rocksdb::DB;
-use std::time::Duration;
-use tokio::sync::mpsc;
-use tokio::time;
-use tracing::{debug, info};
+
+use self::errors::FailedToFetchData;
+use self::fetchers::{
+    fetch_block, fetch_block_by_height, fetch_block_chunks, fetch_latest_block, fetch_outcomes,
+    fetch_state_changes, fetch_status,
+};
+use self::utils::convert_transactions_sir_into_local_receipts;
+use crate::streamer::fetchers::fetch_protocol_config;
+use crate::INDEXER;
+use crate::{AwaitForNodeSyncedEnum, IndexerConfig};
 
 mod errors;
 mod fetchers;
 mod metrics;
 mod utils;
 
+lazy_static! {
+    static ref DELAYED_LOCAL_RECEIPTS_CACHE: Arc<RwLock<HashMap<CryptoHash, views::ReceiptView>>> =
+        Arc::new(RwLock::new(HashMap::new()));
```

Comment:
I am unsure about reasoning to have `Arc` here, please advise me if it's not needed here

Reply:
You don't need it here, you don't clone it anywhere